crimsobot/cogs/mystery.py

```diff
@@ -58,6 +58,88 @@ async def tarot(self, ctx: commands.Context, spread: str = 'ppf') -> None:
         fp, descriptions = tarot.reading(spread)
         await ctx.send('\n'.join(descriptions), file=discord.File(fp, 'reading.png'))
 
+    @commands.command(aliases=['tarotcard'])
+    async def card(self, ctx: commands.Context) -> None:
+        """Inspect a tarot card chosen from the menu for further study."""
+
+        # the suits
+        suit_dict = {0: 'Major arcana', 1: 'Wands', 2: 'Pentacles', 3: 'Cups', 4: 'Swords'}
+        suit_list = []
+        for suit in suit_dict:
+            suit_list.append('{}. {}'.format(suit, suit_dict[suit]))
+
+        # prompt 1 of 2: choose suit
+        embed = c.crimbed(
+            title='Choose a suit:',
+            descr='\n'.join(suit_list),
+            thumb_name='wizard',
+            footer='Type the number to choose.'
+        )
+        prompt_suit = await ctx.send(embed=embed)
+
+        # define check for suit
+        def suit_check(msg: discord.Message) -> bool:
+            try:
+                valid_choice = 0 <= int(msg.content) <= 4
+                in_channel = msg.channel == ctx.message.channel
+                is_author = msg.author == ctx.message.author
+
+                return valid_choice and in_channel and is_author
+
+            except ValueError:
+                return False
+
+        # wait for user to spcify suit
+        msg = await self.bot.wait_for('message', check=suit_check, timeout=20)
+
+        if msg is None:
+            await prompt_suit.delete()
+            return
+        else:
+            suit_choice = int(msg.content)
+            await msg.delete()
+            await prompt_suit.delete()
```

Comment:
nit: I'm a fan of returning early in situations like this ([see this for examples](https://blog.timoxley.com/post/47041269194/avoid-else-return-early)). Makes the code flatter and cleaner IMO.

```suggestion
        await prompt_suit.delete()

        if msg is None:
            return

        suit_choice = int(msg.content)
        await msg.delete()
```